kratos/solving_strategies/strategies/explicit_solving_strategy.h

```diff
@@ -0,0 +1,719 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+//
+
+#if !defined(KRATOS_EXPLICIT_SOLVING_STRATEGY)
+#define KRATOS_EXPLICIT_SOLVING_STRATEGY
+
+/* System includes */
+
+/* External includes */
+
+/* Project includes */
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "utilities/parallel_utilities.h"
+#include "solving_strategies/builder_and_solvers/explicit_builder_and_solver.h"
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/** @brief Explicit solving strategy base class
+ * @details This is the base class from which we will derive all the explicit strategies (FE, RK4, ...)
+ */
+template <class TSparseSpace, class TDenseSpace>
+class ExplicitSolvingStrategy
```

Comment:
Why doesn't this derive from `SolvingStrategy`?

The interface is pretty much the same

Reply:
The base solving strategy is templated with the linear solver type, which is not required in the explicit case. We thought that having such useless template argument was a non-sense.

Having said this, I think that the proper way to implement this would be to have a base strategy and then an implicit and explicit derived ones.

Reply:
What are the other differences except the missing linear solver?

Reply:
Right now, I'd say that the API is rather similar.